src/main/scala/diplomaticobjectmodel/OMLogicalTree.scala

```diff
@@ -0,0 +1,72 @@
+// See LICENSE.SiFive for license details.
+
+package freechips.rocketchip.tile
```

Comment:
This should be in package `diplomaticobjectmodel`

Reply:
fixed